com.unity.render-pipelines.core/Runtime/RenderPipeline/IShaderVariantSettings.cs

```diff
@@ -5,11 +5,14 @@ namespace UnityEngine.Rendering
     /// </summary>
     public enum ShaderVariantLogLevel
     {
-        /// <summary>Disable all log for Shader Variant</summary>
+        // <summary>Disable all log for Shader Variant</summary>
+        [Tooltip("No shader variants are logged")]
         Disabled,
-        /// <summary>Only logs SRP Shaders when logging Shader Variant</summary>
+        /// <summary>Only logs Shaders with a Subshader containing the tag RenderPipeline</summary>
+        [Tooltip("Only shaders that are compatible with SRPs (ie: URP, HDRP) are logged")]
```

Comment:
```suggestion
        [Tooltip("Only shaders that are compatible with SRPs (e.g., URP, HDRP) are logged")]
```

i.e. is for _id est_ which is "that is".
e.g. is for "example given".
While i.e. imply the full explanation covering all case, e.g. show only some example.
Here we should also cover Custom RP case. If the system cannot be used with CustomRP, keep i.e. but I guess this is not the case.

Also it seems that we use the coma and not the double dot for that nowadays in American English.